### ISSUE TITLE
Color Mask Controls in Final Cut Pro has no Accessibility Objects in the Viewer

### ISSUE THREAD

**Issue by latenitefilms** [CommandPost/FinalCutPro#48]
**Submitted to Feedback Assistant:** FB6158831

The mask controls in the viewer in Final Cut Pro 10.4.6 currently do not have any Accessibility objects. There are also controls for the masks in the Final Cut Pro Inspector, so currently the only way to control the size, position, rotation and feather of the mask is with the mouse, which is not ideal for accessibility purposes.

I propose that the mask controls should have Accessibility objects in the viewer, and there should also be  Position (X, Y), Scale (All, X, Y), Rotation and Feather controls in the Shape Mask section of the Color Inspector.
